ocs_ci/ocs/perftests.py

```diff
@@ -1023,3 +1027,114 @@ def cleanup_testing_pod(self):
         except Exception:
             log.warning("The POD failed to delete")
             pass
+
+    def get_performance_metrics(self, start_time):
+        # This function is used to determine CPU and memory utilization of openshift storage pods and nodes
+        # start_time (secs) : start time of the test
+        from ocs_ci.ocs.resources import pod as pod1
```

Comment:
Could you please move the import to the top of the file? 
we use import within a method only in case of circular dependency. If that is the case, please add a comment for the same